### PR TITLE
Introduces UMD headers to the web/ folder.

### LINKED ISSUE #6776: Add UMD headers to the web/ files.
We need to add UMD headers for the files we already packaging into pdf_viewer.js. For now we want to refer AMD "pdfjs/" modules directly in their headers, but eventually (after #6775) we need to refer pdf.js.

Let's choose "pdfjsViewer/" or "pdfjsviewer/" as prefix for AMD module names (for files located at web/)

Blocked #6771

/cc @Snuffleupagus

### PR DESCRIPTION
**Pull request by yurydelendik**
Fixes #6776

### CODE REVIEW

**Comment by yurydelendik** [comment]
TODOs:
- [ ] fix tests (unit and reference)
- [ ] verify for the extensions
- [ ] setup dependency between viewer and core module (atm they can be loaded in parallel)

**Comment by yurydelendik** [comment]
Dependo diagram:

screen shot 2016-04-08 at 2 53 14 pm

File: web/app.js [hunk 1]
```diff
@@ -12,18 +12,93 @@
...
+      root.pdfjsWebPDFThumbnailViewer, root.pdfjsWebSecondaryToolbar,
+      root.pdfjsWebPasswordPrompt, root.pdfjsWebPDFPresentationMode,
+      root.pdfjsWebPDFDocumentProperties, root.pdfjsWebHandTool,
+      root.pdfjsWebPDFViewer, root.pdfjsWebPDFRenderingQueue,
+      root.pdfjsWebPDFLinkService, root.pdfjsWebPDFOutlineViewer,
+      root.pdfjsWebOverlayManager, root.pdfjsWebPDFAttachmentViewer,
+      root.pdfjsWebPDFFindController, root.pdfjsWebPDFFindBar,
+      root.pdfjsWebMozPrintCallbackPolyfill, root.pdfjsWebPDFJS);
+  }
+}(this, function (exports, uiUtils, firefoxCom, downloadManagerLib,
```
Comment (Snuffleupagus) [inline web/app.js]: As I mentioned on IRC, the fact that only some of these have `Lib` at the end confused me initially.
So I'm wondering if we could append `Lib` consistently on *all* of these, to reduce possible confusion here?

**Comment by Snuffleupagus** [comment]
In my testing this code seem to work exactly the same way as before, in both the viewer/addons.
Hence I'm thinking that we can simply land this (pending an answer/fix to my question above), and then deal with any potential fallout in a follow-up (to avoid bit-rotting this PR).

File: web/app.js (hunk 1 above)
Comment (yurydelendik) [inline web/app.js]: "Lib" suffix was added

**Comment by timvandermeij** [comment]
Awesome work!

**Pull request merged**